components/dev_to/dev_to.app.mjs

```diff
@@ -0,0 +1,40 @@
+import axios from "@pipedream/platform";
+
+export default {
+  type: "app",
+  app: "dev_to",
+  methods: {
+    _getBaseUrl() {
+      return "https://dev.to/api";
+    },
+    _getHeaders() {
+      return {
+        "Content-Type": "application/json",
+        "api-key": `${this.$auth.api_key}`,
+      };
+    },
+    _getAxiosParams(opts = {}) {
+      const res = {
+        ...opts,
+        url: this._getBaseUrl() + opts.path,
+        headers: this._getHeaders(),
+      };
+      return res;
+    },
+    async _makeRequest(opts) {
+      return axios(this._getAxiosParams(opts));
```

Comment:
Humm, normally we need to pass a context as the first param here. The actions have a property called `$`, you can use it as the first param, and you can also use `this` if it's not possible to use  `$`.
Please take a look on `blogger/create-post` and `blogger.app.mjs`

Reply:
@vellames-turing whew so many things to remember! Thanks, I just pushed an update to include the context.